library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/processor/WireframeUtils.kt

```diff
@@ -86,7 +86,7 @@ internal class WireframeUtils {
     private fun MobileSegment.ShapeStyle.isOpaque(): Boolean {
         return this.opacity == FULL_OPACITY_ALPHA &&
             this.backgroundColor != null &&
-            this.backgroundColor.takeLast(2) == FULL_OPACITY_ALPHA_AS_HEXA_STRING
+            this.backgroundColor.takeLast(2).matches(FULL_OPACITY_REG_EX)
```

Comment:
probably we don't need regex here, we can just convert to upper/lowecase and compare?

Reply:
yes was not sure which one is better but I think conversion is faster. I will do that.